src/smartdc/lib/ntp_config.js

```diff
@@ -199,9 +205,16 @@ generate_file(servers, allowed_subnets, trusted, orphan)
 				process.exit(1);
 			}
 
-			out.push(mod_util.format('%s %s mask %s',
-			    fam === 4 ? 'restrict' : 'restrict -6', as[0],
-			    mask));
+			/*
+			 * We should deduplicate these networks since the -a
```

Comment:
Is this a "We don't do it now, but we should," sort of comment?  If so, please mark with TODO:

```
/*
 * TODO: We should deduplicate...
```

If this code below actually does the dedup (doesn't *seem* to but I might be missing something), lose the "should" as we're actually deduplicating:

```
/*
 * We deduplicate these networks...
```

Reply:
Clarified this comment.